server/server.js

```diff
@@ -3,7 +3,7 @@ import { Meteor } from 'meteor/meteor';
 
 /* eslint-enable import/no-unresolved */
 
-import { determineStatus, UserPresence } from './user-presence.js';
+import UserPresence, { determineStatus } from './user-presence.js';
```

Comment:
UserPresence would be undefined in this case since `user-presence.js` exports it as a named export

Reply:
Darn! Missed that! Fixed!